linux/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+install(FILES org.qgis.qgis.desktop DESTINATION share/applications)
+install(FILES org.qgis.qgis.appdata.xml DESTINATION share/metainfo)
+install(FILES icons/qgis_icon.svg DESTINATION share/icons/hicolor/scalable/apps)
```

Comment:
`qgis_icon.svg` is not in `linux` but `../images` (maybe all this should be moved there instead of `linux/`).  Doesn't work for me.
Also `qgis_icon.svg` is expected to be installed to `/usr/share/pixmaps` (see the original `debian/rules`; otherwise `debian/qgis.install` needs to be adapted too; also applies to other images and other directories).